sdk/core/azure-core/CHANGELOG.md

```diff
@@ -10,6 +10,8 @@
 
 ### Bugs Fixed
 
+- Added timeout in aiohttp.close().
```

Comment:
ChangeLog is for customers, and should not explain implementation details, but behavior

```suggestion
- Reduce risk of hanging while closing if server do not follow best practices. Fix #41363
```